data-model/annotations/schema/annotation.json

```diff
@@ -161,27 +246,48 @@
     },
     "dcterms:created": {
       "type": "string",
-      "format": "date-time"
+      "format": "date-time",
+      "description": "The date and time when the annotation was created. http://purl.org/dc/terms/created"
     },
     "ods:deletedOn": {
       "type": "string",
-      "format": "date-time"
+      "format": "date-time",
+      "description": "Filled if the annotation was archived. The date and time when the annotation was archived"
     },
     "oa:generated": {
       "type": "string",
-      "format": "date-time"
+      "format": "date-time",
+      "description": "The date and time when the annotation was generated. http://purl.org/dc/terms/issued",
+      "$comment": "Be aware this is not a valid oa field. This is called generated within W3C annotation model but actually maps to dcterms:issued. We kept it generated to keep in line with W3C annotation standard."
     },
     "as:generator": {
       "type": "object",
+      "description": "Object containing information on who generated the object. Generated is here seen as who stored/indexed the object. In most case this will be the annotation-processing-service",
       "properties": {
-        "ods:id": {
-          "type": "string"
+        "ods:type": {
+          "type": "string",
+          "description": "The type of the creator.",
```

Comment:
We have created a vocab for those yet. It will be the handle of the types of DS, DM and annotation but those types are not yet fixed. Left it String when it would probably change. We can fix those to enums later